recipe/meta.yaml

```diff
@@ -12,20 +13,28 @@ source:
   patches:
     - glib-mkenums-wrapped.win.patch  # [win]
     - exportsymbols.patch  # [win]
-    # system headers must be included before jpeg on Windows
     - windows-data.patch  # [win]
     # libtiff uses Unix I/O even on Windows
     # https://github.com/conda-forge/libtiff-feedstock/pull/51
     - disable-libtiff-win32-io.patch  # [win]
 
 build:
-  number: 1
+  number: 2
```

Comment:
Because https://github.com/conda-forge/poppler-feedstock/pull/117 was also just merged (which bumped the build number from 1 to 2 as well), I think this PR effectively didn't bump the build number.

(I don't know if this is the reason, but I was looking at this feedstock because of having problems with libdeflate -> libtiff -> poppler)

Reply:
I reported the issue I was looking into at GDAL: https://github.com/conda-forge/gdal-feedstock/issues/537 (but to repeat, might not be related at all, I just was looking at related feedstocks to check for recent updates that might have triggered the broken install)

Reply:
Thanks for pointing this out. I'll bump the build number again just to be safe.